src/Iot.Device.Bindings/Iot.Device.Bindings.csproj

```diff
@@ -18,6 +18,11 @@
     <Compile Include="$(DeviceRoot)**/*.cs" Exclude="@(_ExcludeFromCompile)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Include="Utils\BitmapImage.cs" />
```

Comment:
As per my other comment, these two files shouldn't leave here so please remove these edits. Once you move the files to the place I suggested, then they will be automatically compiled by this project.